calico/observability/view-flow-logs.mdx

```diff
@@ -3,4 +3,98 @@ description: View flow logs in the Calico Whisker web console.
+| `dest_port`                       | long              | Destination port. Not applicable for ICMP packets.                                                                                                                                                                                                                                                                                                   |
+| `dest_labels`                     | array of keywords | Labels applied to the destination pod. A hyphen indicates aggregation.                                                                                                                                                                                                                                                                               |
+| `reporter`                        | keyword           | - `src`: flow came from the pod that initiated the connection.<br />- `dst`: flow came from the pod that received the initial connection.                                                                                                                                                                                                            |
+| `packets_in`                      | long              | Number of incoming packets since the last export.                                                                                                                                                                                                                                                                                                    |
+| `packets_out`                     | long              | Number of outgoing packets since the last export.                                                                                                                                                                                                                                                                                                    |
+| `protocol`                        | keyword           | Protocol.                                                                                                                                                                                                                                                                                                                                            |
+| `policies`                        | array of arrays   | List of policies that interacted with this flow. See [Format of the policies field](#format-of-the-policies-field).                                                                                                                                                                                                                                                |
```

Comment:
guessing `format-of-the-policies-field` is a WIP?

Reply:
Holdover from CE, now removed.